routing/async_router.hpp

```diff
@@ -114,6 +116,7 @@ class AsyncRouter final
   m2::PointD m_startPoint;
   m2::PointD m_finalPoint;
   m2::PointD m_startDirection;
+  bool m_adjust = false;
```

Comment:
You need to initialize m_startPoint, m_finalPoint and m_startDirection too, right?